index.js

```diff
@@ -319,7 +320,7 @@ async function connect(client, requeue, gamemode) {
 		if (!packet.canConnect) {
 			client.end(packet.errorMessage ?? "Disconnected");
 		}
-		if (requeue) return;
+		if (requeue) {return;}
```

Comment:
```suggestion
		if (requeue) return;
```
gahhh this must have been a formatter, right?